kubernetes-manifests/server.deployment.yaml

```
@@ -32,7 +32,7 @@ spec:
         - name: ${deployment_name}
           image: ${image_name}
           imagePullPolicy: Always
-          % if address_type != "ipv6":
+          % if address_type != "ipv6": # TODO test without this when ipv6 HC fw rules are in place
```

uses `##` notation so it's not rendered in the final yaml: https://docs.makotemplates.org/en/latest/syntax.html#comments

```suggestion
          ## TODO:(lsafran) test without this when ipv6 HC fw rules are in place
          % if address_type != "ipv6":
```